bin/spotxBidAdapter.md

```diff
@@ -30,7 +30,47 @@ This adapter requires setup and approval from the SpotX team.
                 outstream_options: { // Needed for the default outstream renderer. Fields video_slot/content_width/content_height are mandatory
                     video_slot: 'video1',
                     content_width: 300,
-                    content_height: 250
+                    content_height: 250,
+                    custom_override: [{ // This option is not mandatory though used to override default renderer parameters using EASI player options in here: https://developer.spotxchange.com/content/local/docs/sdkDocs/EASI/README.md
+                        name: 'content_width',
+                        value: 640
+                    }, {
```

Comment:
let's pare this down to just an example or two, rather than spelling out all of them